source/contracts/reporting/DisputeBondToken.sol

```diff
@@ -38,14 +40,19 @@ contract DisputeBondToken is DelegationTarget, Typed, Initializable, ERC20Basic,
         IReputationToken _reputationToken = getReputationToken();
         uint256 _amountToTransfer = _reputationToken.balanceOf(this);
         bondRemainingToBePaidOut = bondRemainingToBePaidOut.sub(_amountToTransfer);
+        if (bondRemainingToBePaidOut < bondAmount) {
+            uint256 _amountToDeductFromExtraPayout = _amountToTransfer.min(bondAmount - bondRemainingToBePaidOut);
+            getUniverse().decreaseExtraDisputeBondRemainingToBePaidOut(_amountToDeductFromExtraPayout); 
+        }
         _reputationToken.transfer(bondHolder, _amountToTransfer);
         return true;
     }
 
-    // FIXME: We should be minting coins in this scenario in order to achieve 2x target payout for bond holders during a fork.  Ideally, the amount minted is capped at the amount of tokens redeemed on other universes, so we may have to require the user to supply universes to deduct from with their call to this.
+    // NOTE: The UI should warn users about doing this before REP has migrated fully out of the forking universe in order to get their extra payout, which is sourced from REP migrated to other Universes. There is no penalty for waiting (In fact waiting will only ever get them a greater payout)
```

Comment:
This comment (and implementation I think) aren't quite right.  You only get the 5% bonus REP if you call this function during the fork (not after the fork is over).  This means that there _is_ a penalty (opportunity cost) for migrating late.

Looking over the code, I would expect to see a `if (forkingMarket.isFinalized)` statement somewhere but I don't.  Perhaps I'm missing it?  If not, then that is necessary for calculating the % bonus.

I believe the 2x bonus should apply even if you migrate after the fork is finalized.

Reply:
The 5% bonus is applied in the ReputationToken as it applies to all migration of REP. The comment is just referring to the 2x bonus.

Reply:
More on that too since it's kind of interesting. I did it this way so there was a balance between encouraging migration of dispute bonds during the fork but also not forgoing potential payouts from REP that may later be migrated to the disputed universe. This way there is a theoretical benefit to the bond holder whether they wait or not, in particular because the 5% bonus will count toward the desired 2x payoff, so people cashing in earlier take less from the pool of REP migrated to the disputed universe.

Reply:
Is it possible for a dispute bond holder to force StakeTokens to be redeemed in their respective forks?  There may be value in allowing anyone to force migrate stake so that the dispute bond holders can force other people's REP to migrate before they call this function.  Alternatively, we could include staked-but-not-migrated REP in the available-to-mint calculation, though that may result in making the accounting more complicated/difficult.

Reply:
Also of note, the 3 dispute bond holders will be competing with each other for what may be a limited pool of REP available for minting.  It would be nice if we could allocate REP to the smaller bonds first, bigger bonds second but I'm worried that would make the accounting more difficult and not really provide significant value.  Thoughts?

Reply:
The issue with the forcing migration function is that the REP holder may select a different universe as right now this only matches with the disputed universe.

Regarding the 3 bonds the calculation right now is based on the percentage of "extra" payout the bond needs relative to all other bonds in that universe, so it should be proportional. While it doesn't favor the smaller bonds it does at least attempt to do something reasonably fair.

This logic existsi n the `mintRepTokensForBondMigration` function